packages/dao/contracts/Api3Template.sol

```diff
@@ -1,9 +1,13 @@
 //SPDX-License-Identifier: MIT
 pragma solidity 0.4.24;
 
-import "./Api3BaseTemplate.sol";
+import "@aragon/templates-shared/contracts/BaseTemplate.sol";
+import "./utils/Api3Voting.sol";
+
+contract Api3Template is BaseTemplate {
+
+    bytes32 constant internal API3_VOTING_APP_ID = 0x9fa3927f639745e587912d4b0fea7ef9013bf93fb907d29faeab57417ba6e1d4;
```

Comment:
Are these chain-agnostic?

Reply:
Unfortunately no, it is not, I've updated contract, so right now it is possible to see which `ID` are we using for rinkeby/mainnet, and which we are using for tests. This is due to the problems with creating subdomains using current procedure for testing, because on rinkeby we are going to use `api3voting.open.aragonpm.eth`, and for testing we are forced to use `api3voting.aragonpm.eth` because we have problems with creating subdomains..

Reply:
Okay I think I know what you're talking about